app/src/androidTest/java/com/okta/idx/android/SelfServicePasswordRecoveryTest.kt

```diff
@@ -47,23 +43,17 @@ class SelfServicePasswordRecoveryTest {
         private const val ERROR_TEXT_VIEW = "com.okta.idx.android:id/error_text_view"
         private const val ID_TOKEN_TYPE_TEXT_VIEW = "com.okta.idx.android:id/token_type"
         private const val CODE_EDIT_TEXT = "com.okta.idx.android:id/code_edit_text"
-        private const val CONFIRMED_PASSWORD_EDIT_TEXT = "com.okta.idx.android:id/confirmed_password_edit_text"
+        private const val CONFIRMED_PASSWORD_EDIT_TEXT =
```

Comment:
We have same ids in some layouts e.g. username_text_input_layout in form_forgot_password and form_username_and_password, wouldn`t that be a problem?

Reply:
Not a problem per se. But would be better to consolidate them. I'll do it in a follow up.